### PR TITLE
[ORDER] Create docker and folder structure for order service

### PR DESCRIPTION
**Pull request by aacgn**
Solves #21

### CODE REVIEW

File: order-service/src/app.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import 'reflect-metadata';
+import DatabaseConfig from './config/database';
+import * as Express from 'express';
+import * as bodyParser from 'body-parser';
+
+class Core {
```
Comment (viniciusgiles) [inline order-service/src/app.ts]: We'll really be using this name? `OrderService` will conflict with `Order` service... Maybe `OrderApp`?

File: order-service/src/config/database.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            }
+        }
+    }
+
+    /**
+     * Gets repository for the given entity class.
+     *
+     * @param entity
+     */
+    public static getRepository<T>(entity: ObjectType<T> | EntitySchema<T> | string): Repository<T> {
```
Comment (viniciusgiles) [inline order-service/src/config/database.ts:27]: I hate to see this duplicated code... I wonder if we really need this function.

File: order-service/src/config/database.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        return getRepository(entity, DatabaseConfig.connectionName);;
```
Comment (viniciusgiles) [inline order-service/src/config/database.ts]: extra ; at the end of the line.

Comment (aacgn) [reply order-service/src/config/database.ts]: Done

File: order-service/src/config/database.ts (hunk 2 above)
Comment (aacgn) [reply order-service/src/config/database.ts:27]: Maybe another time.

File: order-service/src/app.ts (hunk 1 above)
Comment (aacgn) [reply order-service/src/app.ts]: Done

**Review by eduardosm7 — APPROVED** [review]

**Review by viniciusgiles — APPROVED** [review]

**Pull request merged**